README.md

```diff
@@ -8,19 +8,27 @@ Onboard backend project to get familiar with Taqtile's workflow and projects bes
 
 To run the server, you need first to have node.js installed on your machine. After that, you can follow the steps bellow:
 
-### Install dependencies
+### 1. Environment
+
+Edit `sample.env` file to your liking and then rename it to `.env`. After that you can startup the required containers executing the following command:
```

Comment:
Acho que pra ficar mais claro, seria legal instruir a pessoa a copiar o `sample.env` para o `.env` e alterar o `.env` só né? Senão pode ser que haja commits modificando desnecessariamente o `sample.env`.

Num contexto de projeto real tbm, seria um nice-to-have tbm instruir a atualizar o `sample.env` sempre que for acrescentada uma variável de ambiente nova. Faz sentido?

Reply:
Faz sentido!
Eu fiz algumas modificações no README.md para tentar deixar mais alinhado com o que você recomendou (568d5b488a4a7be1524befb04c973eeea9f148b7).